enhancements/storage/csi-driver-operator-merge.md

```diff
@@ -469,6 +469,80 @@ N/A
 
 Same as today.
 
+## Process of moving operators to csi-operator monorepo
+
+We have come with following flow for moving operators from their own repository into csi-operator mono repo.
+
+### Avoid .gitignore related footguns
+
+Remove any .gitignore entries in the source repository that would match a directory / file that we need. For example, `azure-disk-csi-driver-operator` in .gitignore matched `cmd/azure-disk-csi-driver-operator` directory that we really need not to be ignored. See https://github.com/openshift/csi-operator/pull/110, where we had to fix after merge to csi-operator.
+
+### Move existing code into csi-operator repository
+
+Using git-subtree move your existing operator code to https://github.com/openshift/csi-operator/tree/master/legacy
+
+```
+git subtree add --prefix legacy/azure-disk-csi-driver-operator https://github.com/openshift/azure-disk-csi-driver-operator.git master --squash
+git subtree push --prefix legacy/azure-disk-csi-driver-operator https://github.com/openshift/azure-disk-csi-driver-operator.git master
+```
+
+### Add Dockerfiles for building images from new location
+
+Place a `Dockerfile.<operator>` and `Dockerfile.<operator>.test` at top of csi-operator tree and make sure that you are able to build an image of the operator from csi-operator repository.
+
+### Update openshift/release to build image from new location
+Make a PR to openshift/release repository to build the operator from csi-operator. For example - https://github.com/openshift/release/pull/46233.
+
+1. Update also `storage-conf-csi-<operator>-commands.sh`, the test manifest will be at a different location.
+2. Make sure that rehearse jobs for both older versions of operator and newer versions of operator pass.
```

Comment:
Is there a better way to search for rehearsal job names other than looking at jobs in PRs and history? Maybe we could expand on this step a bit.

Reply:
I think when you do `/pj-rehearse` it doe list the jobs and then you can choose and run.